tests/python/pants_test/backend/python/tasks/python_task_test_base.py

```diff
@@ -65,9 +65,11 @@ def create_python_binary(self, relpath, name, entry_point, dependencies=(), prov
         {dependencies}
       ],
       {provides_clause}
+      {shebang_clause}
     )
     """).format(name=name, entry_point=entry_point, dependencies=','.join(map(repr, dependencies)),
-                provides_clause='provides={0},'.format(provides) if provides else ''))
+                provides_clause="provides={0},".format(provides) if provides else '',
```

Comment:
Consider `'` instead of `"` to match else clause.

Reply:
Those two ended up with ```"``` instead of ```'``` to have some sort of similarity based on the need for ```"``` with shebang_clause. However, since shebang_clause can be had without ```"```, I've moved both back over to ```'```.